docs/snippets/getting-started/anta_nrfu_json.sh

```diff
@@ -0,0 +1,9 @@
+anta nrfu \
+    --username arista \
+    --password $LABPASSPHRASE \
```

Comment:
Since it's a getting started should we add a small comment about the env here?

Reply:
We could add something like that similar to the atd-avd repository:

---

Because the password is auto-generated, you must update the ansible_password variable. We will read a configuration file to set our credential file as an environment variable.

Run the following commands from the Programmability IDE terminal:

```bash
export LABPASSPHRASE=`cat /home/coder/.config/code-server/config.yaml| grep "password:" | awk '{print $2}'`
```

Reply:
okay in fact I will remove this alltogether. People can see how to use ANTA_PASSWORD later.